pkg/streamingpromql/engine_fuzz_test.go

```diff
@@ -118,12 +118,12 @@ func testInstantQueries(t *testing.T, startT time.Time, query string, testEnviro
 
 	// if the Prometheus engine can not prepare the query then we expect the MQE to also fail
 	if prometheusError != nil {
-		require.Errorf(t, mqeError, "Prometheus' engine returned error %v, but MQE returned no error", prometheusError)
+		require.Errorf(t, mqeError, "Failed to create query – Prometheus' engine returned error, but MQE returned no error.\nQuery - %q", query)
```

Comment:
[nit] We tend to use colons in other places like this:

```suggestion
		require.Errorf(t, mqeError, "Failed to create query: Prometheus' engine returned error, but MQE returned no error.\nQuery: %q", query)
```

(same applies below)